spec/config/karma/config.common.js

```diff
@@ -40,6 +40,11 @@ module.exports = {
   singleRun: true,
   client: {
     captureConsole: false,
+    jasmine: {
+      // This new behaviour in jasmine is enabled by default but, some of our
```

Comment:
This was breaking some tests because we have some cases that expect a flag to be set by previous case. Unfortunately some stuff in the tests rely on how Jasmine used to work.

I like the idea of adding TODO to refactor those cases. So I'll do that.